kornia/augmentation/container/image.py

```diff
@@ -1,20 +1,22 @@
+import warnings
 from collections import OrderedDict
 from itertools import zip_longest
 from typing import Any, Iterator, List, NamedTuple, Optional, Tuple, Union
 
 import torch
 import torch.nn as nn
 
-from kornia.augmentation.base import _AugmentationBase
+from kornia.augmentation.base import _AugmentationBase, MixAugmentationBase, TensorWithTransMat
 
 __all__ = ["ImageSequential"]
 
 
 class ParamItem(NamedTuple):
     name: str
-    data: Union[dict, list]
+    data: Optional[Union[dict, list]]
```

Comment:
when data can be None ?

Reply:
When we only want to create the operation without specifying the parameter.